music21/instrument.py

```diff
@@ -2095,11 +2095,27 @@ def fromString(instrumentString):
 
     Use "H" or "b-natural" to get an instrument in B-major.  Or donate one to me
     and I'll change this back!
+
+
+    Finally, standard abbreviations are acceptable:
+
+    >>> t10 = instrument.fromString('Cl in B-flat')
+    >>> t10
+    <music21.instrument.Clarinet 'Cl in B-flat'>
+    >>> t9.transposition
+    <music21.interval.Interval M-2>
+
+    This should work with or without a terminal period (for both 'Cl' and 'Cl.'):
+
+    >>> t11 = instrument.fromString('Cl. in B-flat')
+    >>> t10.__class__ == t10.__class__
```

Comment:
ah, we lost `t11`, didn't we?  even easier to read would be `isinstance(t11, instrument.Clarinet)`

Reply:
Thanks! I'll fix that typo and one in the test before. Very happy for the style of this testing run to change, though for consistency's sake many of the existing test above should also be edited in that case.